src/App.js

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
 
 import './App.scss';
+import PostList from './components/PostList/PostList';
 
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
+const postsData = posts.map(post => ({
```

Comment:
data - надто загальна назва
```suggestion
const preparedPosts = posts.map(post => ({
```